course_content/notebooks/numpy_intro.ipynb

```diff
@@ -113,6 +99,8 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "#### Exercise\n",
```

Comment:
@LukeC92 has made some changes to the numbering of the exercises in PR #136.
I would suggest changing this to "Exercise 1" and merging this PR before #136.

Reply:
If we do that we'll need to make sure **all** the exercises have numbers. As there are now _many_ exercises in the NumPy course (perhaps too many!) I'm not overly keen to add numbering. 

Also, we shouldn't let another PR hold up this PR. We can just rebase later PRs to make sure all the dust settles in the right place.